modules/rosa-sdpolicy-networking.adoc

```diff
@@ -34,7 +34,7 @@ Starting with {product-title} 4.14, the Custom Domain Operator is deprecated. To
 == Load balancers
 
 ifdef::rosa-with-hcp[]
```

Comment:
@laubai Out of scope, but I still want to provide this suggestion so that you can track it and fix it when time permits, before L11. This module needs the :_mod-docs-content-type: <TYPE>  metadata. Please refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Reply:
Thanks Srivaralakshmi! Out of scope for this minor typo fix but I'm raising this in ROSA docs chat to verify we have this in all of our modules.